ete2/parser/newick.py

```diff
@@ -55,6 +55,10 @@
 #_NAME_RE = "[^():,;\[\]]+"
 _NAME_RE = "[^():,;]+?"
 
+"thanks to: http://stackoverflow.com/a/29452781/1006828"
+_QUOTED_TEXT_RE = r"""((?=["'])(?:"[^"\\]*(?:\\[\s\S][^"\\]*)*"|'[^'\\]*(?:\\[\s\S][^'\\]*)*'))"""
+_QUOTED_TEXT_PREFIX='ete2_quotref_'
```

Comment:
Use the form `VAR = value` instead of `VAR=value` and keep it consistent.
Also applies to the rest of the patch where a mix of both styles were used.